docker-compose-services/redis/README.md

```diff
@@ -60,27 +60,70 @@ $ ddev describe
 │          │      │ Host: localhost:54710                   │                       │
 ```
 
-## Examples
+This example shows:
 
-Frameworks, such as Laravel, require configuration to use the redis service.
+* From the webserver point-of-view:
+  * REDIS HOST: ddev-example-redis
```

Comment:
May I suggest to replace this with `ddev-${DDEV_SITENAME}-redis` instead? 

...to emphasize what part of the name is dynamically defined.